File: PROGRAMACION/EJERCICIOS/CUADRADO Y CUBO/src/ejercicio10/Ejercicio10.java
/*
 * Click nbfs://nbhost/SystemFileSystem/Templates/Licenses/license-default.txt to change this license
 * Click nbfs://nbhost/SystemFileSystem/Templates/Classes/Main.java to edit this template
 */
package ejercicio10;
import java.util.Scanner;
/*
Escribe un programa que muestre en tres columnas, el cuadrado y el cubo de
los 5 primeros números enteros a partir de uno que se introduce por teclado.
*/
/**
 *
 * @author Gonzalo
 */
public class Ejercicio10 {

    /**
     * @param args the command line arguments
     */
    public static void main(String[] args) {
        // TODO code application logic here
        
         int num,cuadrado = 0,cubo;
        
         Scanner sc = new Scanner(System.in);
         System.out.println("Introduce un numero:");    
         num = sc.nextInt();
         
        for (int i = num; i <= num + 5; i++) {
            //System.out.println(i);
            cuadrado = (int) Math.pow(i, 2);
            cubo = (int) Math.pow(i, 3);
            
            System.out.println(cuadrado);
            System.out.print("   ");
            System.out.println(cubo);
            
        }
        
    }
    
}

File: ENTORNOS/PYTHON SECUENCIALES/Ejercicio10.py
#Gonzalo Carretero
#Algoritmo que muestre la tabla de multiplicar de los números 1,2,3,4 y 5.

numero_carretero = 0

for i in range (1 , 6):
    print(" ")
    print("La tabla de",i,"es:")
    print(" ")
    
    numero_carretero = numero_carretero + 1
    for i in range (1 , 10 + 1):
        resultado_carretero = numero_carretero * i
        print(numero_carretero,"x",i,"=",resultado_carretero)
    print (" ")